src/main/js/Upload/FileUploadTable.js

```diff
@@ -0,0 +1,97 @@
+//@flow
+import React from "react";
+import {translate} from "react-i18next";
+import injectSheet from "react-jss";
+import classNames from "classnames";
+import {Subtitle} from "@scm-manager/ui-components";
+
+const styles = {
+  nameColumn: {
+    width: "60%"
+  },
+  noBorderLeft: {
+    "& td:first-child": {
+      borderLeft: "none"
+    }
+  },
+  marginTop: {
+    marginTop: "2rem"
+  }
+};
+
+type Props = {
+  files: File[],
+  removeFileEntry: any => void,
+  disabled: boolean,
+  //context props
+  t: string => string,
+  classes: any
+};
+
+class FileUploadTable extends React.Component<Props> {
+  removeEntry = file => {
+    this.props.removeFileEntry(file);
+  };
+
+  humanFileSize(bytes) {
```

Comment:
We have a component named `FileSize` for this. Would be good if we could use this here. Otherwise the delete button may be hidden.

Reply:
moved fileSize from ui to ui-components and use it in the plugin